modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/db/AppDatabase.kt

```diff
@@ -841,6 +843,10 @@ abstract class AppDatabase : RoomDatabase() {
             }
         }
 
+        val MIGRATION_100_101 = addMigration(100, 101) { database ->
+            database.execSQL("ALTER TABLE episode ADD COLUMN auto_download_limit INTEGER DEFAULT 2")
+        }
```

Comment:
This migration is incorrect.

1. It doesn't update `auto_download_limit` for existing podcasts.
2. The definition doesn't match `Podcast` entity. The entity doesn't define default value on the SQL level but on the Kotlin level so there no `DEFAULT 2` when the table is created without migration.
3. The migration crashes the app due to a typo. It should be `TABLE podcasts` .instead of `TABLE episode`.